routes/user.js

```diff
@@ -239,6 +239,60 @@ router.post("/verify-reset-code", userController.verifyResetCode);
 
 router.post("/reset-password", userController.resetPassword);
 
+/**
+ * @swagger
+ * /user/connections:
+ *  get:
+ *    tags:
+ *    - "user"
+ *    summary: Get all connections for the user that is currently signed in
+ *    security:
+ *    - bearerAuth: []
+ *    responses:
+ *      '200':
+ *        description: A successful response, with array of user's connections
+ *        schema:
+ *          type: array
+ *          items:
+ *            type: object
+ *            properties:
+ *              id:
+ *                type: number
+ *                description: Connection ID
+ *              created_at:
+ *                type: string
+ *                format: date-time
+ *                description: Date and time when the connection request has been sent
+ *              accepted_at:
+ *                type: string
+ *                format: date-time
+ *                description: Date and time when the connection request has been accepted
+ *              user:
+ *                type: object
+ *                properties:
+ *                  id:
+ *                    type: number
+ *                    description: User ID
+ *                  fullname:
+ *                    type: string
+ *                    description: User's full name
+ *                  username:
```

Comment:
Would it be a good idea to return user's profile photo here as well? Depending on where on the frontend this would be called..@fifi98 